sensorlib/src/util/SimulatedSensorUtils.py

```diff
@@ -32,7 +32,7 @@ def default(self, obj):
                 'angularVelocity': {'x': obj.angularVelocity[0], 'y': obj.angularVelocity[1], 'z': obj.angularVelocity[2]},
                 'angularVelocityCovariance': obj.angularVelocityCovariance.tolist(),
                 'size': {'length': obj.size[0], 'height': obj.size[1], 'width': obj.size[2]},
-                'timestamp': (int)(obj.timestamp), #TODO need to fix other instances in cdasim before changing https://github.com/usdot-fhwa-stol/carma-utils/issues/187
+                'timestamp': (int)((obj.timestamp)*1000), #TODO need to fix other instances in cdasim before changing https://github.com/usdot-fhwa-stol/carma-utils/issues/187
```

Comment:
We are planning to fix everywhere this change is affecting all at once, so let's remove the TODO line. It will be resolved by this PR. However, we can mention that "CDASim expects Int in milliseconds"